### DISCUSSION TITLE
Re-order Device Data columns

### DISCUSSION THREAD

**Discussion by kabulNoorBiomed** [Samedis-care/samedis-care-community#79] [Ideas]
It is good having some columns which can be pinned in the device data page. However, I think it would be better if users were able to customise the order of device columns, perhaps by simply dragging the column headings left or right to rearrange them.

**Comment by YvesR** [comment]
Re-Arrange columns is a feature on the road map in general, but I can not say when we are implementing this, as there are many other tasks as well and this pro feature is limited to just few users.
